samples/snippets/src/test/java/com/example/spanner/SpannerSampleIT.java

```diff
@@ -328,6 +328,9 @@ public void testSample() throws Exception {
               "Backup %s on database %s pending:",
               backupId.getName(),
               dbId.getName()));
+      assertThat(out).contains(
```

Comment:
nit: don't use `assertThat`, we should prefer to use `Assert.assert*` methods, such as `Assert.assertTrue("Out does not contain " + ..., out.contains(...))`